mvpa2/support/nibabel/afni_niml.py

```diff
@@ -285,12 +288,33 @@ def rawniml2string(p, form='text'):
         # remove some unncessary fields
         for f in ['vec_typ', 'vec_len', 'vec_num']:
             q.pop(f, None)
+    else:
+        has_body=False
 
     s_name = q.pop('name', None).encode()
     s_header = _header2string(q)
 
-    d = map(lambda x:x.encode(), ['<', '\n', ' >', '</', '>'])
-    return b''.join((d[0], s_name, d[1], s_header, d[2], s_body, d[3], s_name, d[4]))
+    if has_body:
+        delim = ['<', '\n', ' >', '</', '>']
+        values = [s_name, s_header, s_body, s_name]
+    else:
+        delim = ['<', '\n', '/>']
+        values = [s_name, s_header]
+
+    delim_enc=map(lambda x:x.encode(), delim)
```

Comment:
PEP8 spaces here and below ;)  Try  pycharm editor -- it highlights all those nicely